temporalcli/commandsmd/commands.md

```diff
@@ -1375,6 +1515,26 @@ temporal task-queue describe \
     --task-queue-type "activity"`
 ```
 
+This command provides the following task queue statistics:
+- `ApproximateBacklogCount`: The approximate number of tasks backlogged in this
+  task queue.  May count expired tasks but eventually converges to the right
+  value.
+- `ApproximateBacklogAge`: How far "behind" the task queue is running. This is
+  the difference in age between the oldest and newest tasks in the backlog,
+  measured in seconds.
+- `TasksAddRate`: Approximate rate at which tasks are being added to the task
+  queue, measured in tasks per second, averaged over the last 30 seconds.
+  Includes tasks dispatched immediately without going to the backlog
+  (sync-matched tasks), as well as tasks added to the backlog.
+- `TasksDispatchRate`: Approximate rate at which tasks are being dispatched from
```

Comment:
These two metrics can be significantly off from what the user might expect due to eager tasks and sticky queues. I think it's worth to mention something about that and link to the docs page that explains the inaccuracy in more details. Something like: "Not all tasks dispatched to the workers are counted in this metric, for more information please see [docs]."